src/component/App/index.js

```diff
@@ -1,20 +1,30 @@
 import React, { Component } from "react";
 import Game from "../Game";
 import StartGame from "../startGame";
+import GameOver from "../GameOver";
 
 class App extends Component {
   state = {
+    name:"",
     status: -1,
     score: 0
   };
   setStatus = newStatus => {
     this.setState({ status: newStatus });
   };
   incScore = () => {
-    this.setState(({ score }) => {
+    this.setState(() => {
       return { score: this.state.score + 1 };
     });
   };
+resetScore=()=>{
+  this.setState({score:0})
+}
+setName=(name)=>{
+  this.setState({"name": name})
+
```

Comment:
Good job on setting the name by passing the function to the child component :+1: